core/iwasm/fast-jit/fe/jit_emit_function.c

```diff
@@ -157,29 +157,81 @@ check_app_addr_and_convert_draft(JitCompContext *cc, WASMFunction *func_ctx,
     return true;
 }
 
+static bool
+pre_load(JitCompContext *cc, JitReg *argvs, const WASMType *func_type)
+{
+    JitReg value;
+    uint32 i, outs_off;
+    // JitReg argv;
+
+    /* Prepare parameters for the function to call */
+    outs_off =
+        cc->total_frame_size + offsetof(WASMInterpFrame, lp)
+        + wasm_get_cell_num(func_type->types, func_type->param_count) * 4;
```

Comment:
No need to create outs_off and store reg into it, just store the popped reg into argvs[].